CHANGELOG.rst

```diff
@@ -11,9 +11,10 @@ and this project adheres to `Semantic Versioning <https://semver.org/spec/v2.0.0
 =============
 
 Changed
--------
+=======
```

Comment:
Was this an intended changed during the merge?

Reply:
oops, nope